### PR TITLE
Add configs for ttl of Alluxio cache

### PR DESCRIPTION
**Pull request by beinan**
Add support for the TTL of Alluxio SDK cache

```
== RELEASE NOTES ==

Hive Changes
* Add support for the TTL of Alluxio SDK cache
``````

### CODE REVIEW

File: presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java [hunk 1]
```diff
@@ -47,7 +49,10 @@ public void testDefaults()
                 .setTimeoutThreads(64)
                 .setCacheQuotaEnabled(false)
                 .setShadowCacheEnabled(false)
-                .setShadowCacheWindow(new Duration(7, DAYS)));
+                .setShadowCacheWindow(new Duration(7, DAYS))
+                .setTtlEnabled(false)
+                .setTtlCheckInterval(new Duration(1, HOURS))
```
Comment (apc999) [inline presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java:54]: better to keep them alphabetical

```suggestion
                .setTtlCheckInterval(new Duration(1, HOURS))
                .setTtlEnabled(false)
```

File: presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java [hunk 2]
```diff
@@ -68,6 +73,9 @@ public void testExplicitPropertyMappings()
                 .put("cache.alluxio.quota-enabled", "true")
                 .put("cache.alluxio.shadow-cache-enabled", "true")
                 .put("cache.alluxio.shadow-cache-window", "1d")
+                .put("cache.alluxio.ttl-enabled", "true")
+                .put("cache.alluxio.ttl-check-interval", "60s")
```
Comment (apc999) [inline presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java:77]: better to keep them alphabetical

```suggestion
                .put("cache.alluxio.ttl-check-interval", "60s")
                .put("cache.alluxio.ttl-enabled", "true")

```

File: presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java [hunk 3]
```diff
@@ -84,7 +92,10 @@ public void testExplicitPropertyMappings()
                 .setTimeoutThreads(512)
                 .setCacheQuotaEnabled(true)
                 .setShadowCacheEnabled(true)
-                .setShadowCacheWindow(new Duration(1, DAYS));
+                .setShadowCacheWindow(new Duration(1, DAYS))
+                .setTtlEnabled(true)
+                .setTtlCheckInterval(new Duration(60, SECONDS))
```
Comment (apc999) [inline presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java:97]:
```suggestion
                .setTtlCheckInterval(new Duration(60, SECONDS))
                .setTtlEnabled(true)
```

**Review by apc999 — APPROVED** [review]
LGTM modulo minor tweaks here

**Review by zacw7 — APPROVED** [review]
Thanks for the prompt change!

**Review by NikhilCollooru — APPROVED** [review]

**Comment by zacw7** [comment]
Thanks @NikhilCollooru for review.

@beinan could you please fix the checkstyle issues then we can have this change merged? Thanks!

**Pull request merged**